compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -3500,3 +3503,62 @@ OMR::X86::CodeGenerator::getOutOfLineCodeSize()
 
    return totalSize;
    }
+
+void
+OMR::X86::CodeGenerator::moveOutOfLineInstructionsToWarm()
+   {
+   // OOL instructions are already attached at the end of the IL (after cold instructions)
+   // and register allocated.
+   // Move them to immediately after the last warm instruction, unless they already happend to be
+   // there (e.g. there are no cold instructions)
+   //
+   if (self()->comp()->getOption(TR_TraceCG))
+      traceMsg(self()->comp(), "Moving OutOfLine instructions\n");
+
+   auto oiIterator = self()->getOutlinedInstructionsList().begin();
+
+   while (oiIterator != self()->getOutlinedInstructionsList().end())
+      {
```

Comment:
I think choosing more descriptive names for the variables will make this code easier to follow because there are a lot of links changing hands.

Suggestions:

`firstInstruction` -> `firstOLInstruction`
`lastInstruction` -> `lastOLInstruction`
`appendInstruction` - > `mainlineAppendInstruction`
`followInstruction` -> `mainlineFollowInstruction`

Reply:
done